ApiBundle/Controller/ContentController.php

```diff
@@ -113,7 +114,7 @@ public function listAction(Request $request)
         );
 
         $repository =  $this->get('open_orchestra_model.repository.content');
-        $contentCollection = $repository->findByContentTypeInLastVersionForPaginateAndSearch($contentType, $columnsNameToEntityAttribute, $columns, $search, $order, $skip, $limit);
+        $contentCollection = $repository->findByContentTypeInLastVersionForPaginateAndSearchAndSiteId($contentType, $columnsNameToEntityAttribute, $columns, $search, $siteId, $order, $skip, $limit);
```

Comment:
Yes, but contrary to all the search parameters, which are given by  the DataTable request, the siteId is an internal parameter, that's why I added it in the method name